src/settings/dto/update-setting.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { PartialType } from "@nestjs/mapped-types";
+import { CreateSettingDto } from './create-setting.dto';
+
+export class UpdateSettingDto extends PartialType(CreateSettingDto) {}
```

Comment:
Let's allow this to explicitly take name and value, and we only update value in the logic.